src/lib/swapper/swappers/ThorchainSwapper/buildThorTrade/buildThorTrade.ts

```diff
@@ -112,12 +113,17 @@ export const buildTrade = async (
     if (maybeThorTxInfo.isErr()) return Err(maybeThorTxInfo.unwrapErr())
     const { vault, opReturnData } = maybeThorTxInfo.unwrap()
 
-    // TODO(gomes): update to use from instead of wallet
-    const buildTxResponse = await (
-      sellAdapter as unknown as UtxoBaseAdapter<ThorUtxoSupportedChainId>
-    ).buildSendTransaction({
-      value: sellAmountCryptoBaseUnit,
+    if (!wallet) throw new Error('wallet is required to make a trade Tx')
```

Comment:
wallet is required prop, reason for this check?

Reply:
Nice catch, probably a leftover of wallet being optional at some point in the initial PR - removed in https://github.com/shapeshift/web/pull/4827/commits/043646bde09988f733ea0ec760d0c830a440a6d2